test cases/d/9 features/meson.build

```diff
@@ -0,0 +1,28 @@
+project('D Features', 'd')
+dc = meson.get_compiler('d')
+
+# directory for data
+data_dir = join_paths(meson.current_source_dir(), 'data')
```

Comment:
Passing `'data'` should work.

Reply:
In that case I will have to expand the string depending on the context. I will look into that.

Reply:
Okay, the path handling works like the rest of Meson now (including both the path in the build directory and the source directory in case a relative path is given).